app/models/rejection.rb

```diff
@@ -11,4 +11,12 @@ class Rejection < ActiveRecord::Base
   belongs_to :visit
 
   validates :reason, inclusion: { in: REASONS }
+
+  def pvo_possible?
```

Comment:
A fair point… Although I might argue that the staff would have that domain knowledge but people working on the codebase might not initially?

Simplistically, `vo` could just be substituted with `visit_order`… But I have no idea what the `p` in `pvo` stands for. Maybe @kyliemulholland knows?